### DISCUSSION TITLE
Running 2 apps in production with a turbo command?

### DISCUSSION THREAD

**Discussion by baptisteArno** [vercel/turborepo#1267] [Help] [answered]
Is it possible to run apps in production with the turbo CLI? When I try with 2 Next.js apps, I get this runtime error on page load for both apss:

```
Cannot find module '<path_to_app_1>/node_modules/react/jsx-runtime.js'
```

**Comment by anthonyshew** [comment, marked as answer]
Turborepo is not a runtime task runner. We help you run tasks in CI.
